src/main/java/com/checkmarx/flow/controller/GitHubController.java

```diff
@@ -164,7 +183,7 @@ public String pingRequest(
             String gitUrl = Optional.ofNullable(pullRequest.getHead().getRepo())
                     .map(Repo::getCloneUrl)
                     .orElse(repository.getCloneUrl());
-            String token = scmConfigOverrider.determineConfigToken(properties, controllerRequest.getScmInstance());
+            String token = gitHubAuthService.getBasicAuth(controllerRequest.getScmInstance());
```

Comment:
this will disable the config override, we need to see how to support both logics

Reply:
Currently the process in auth service is:   if there is a github apps config, then use it, else use the config overrider. 

For me a github app is organization wide, so is there a point to override it ?   
NOTE: currently the config reference a single github app installed on a single organization. if somebody want to use this against multiple orgs, he may need to update the apps selection.